### PR TITLE
Member Audit

### PR DESCRIPTION
**Pull request by damacus**
Remove inactive members

### CODE REVIEW

File: modules/org_membership/main.tf [hunk 1]
```diff
@@ -354,22 +191,7 @@ resource "github_membership" "MichaelLopez" {
...
-  role     = "member"
-}
-
 resource "github_membership" "mthssdrbrg" {
   username = "mthssdrbrg"
   role     = "member"
 }
-
-resource "github_membership" "adriancohea" {
-  username = "AdrianCohea"
```
Comment (JohnRoesler) [inline modules/org_membership/main.tf]: They just joined 2 days ago

File: modules/org_membership/main.tf [hunk 2]
```diff
@@ -259,71 +156,11 @@ resource "github_membership" "sinfomicien" {
...
-  username = "mohitsethi"
-  role     = "member"
-}
-
-resource "github_membership" "johnbellone" {
-  username = "johnbellone"
-  role     = "member"
-}
-
-resource "github_membership" "xorima" {
```
Comment (JohnRoesler) [inline modules/org_membership/main.tf]: don't remove Jason! 😨

Comment (damacus) [reply modules/org_membership/main.tf]: hahga oops :D

File: modules/org_membership/main.tf (hunk 1 above)
Comment (damacus) [reply modules/org_membership/main.tf]: They've been added twice as two different accounts 🤔 We should probably only have one.

File: modules/org_membership/main.tf (hunk 2 above)
Comment (JohnRoesler) [reply modules/org_membership/main.tf]: still seeming him removed?

Comment (JohnRoesler) [reply modules/org_membership/main.tf]: nvm, just moved 👍

File: modules/org_membership/main.tf [hunk 3]
```diff
@@ -1,13 +1,3 @@
-resource "github_membership" "atomic-penguin" {
-  username = "atomic-penguin"
-  role     = "member"
-}
-
-resource "github_membership" "TD-4242" {
```
Comment (JohnRoesler) [inline modules/org_membership/main.tf]: this person is a maintainer on https://github.com/sous-chefs/zabbix-agent/blob/35dd693dc710a65c1cfd33c5f2219eea394468db/zabbix-agent.tf

File: modules/org_membership/main.tf [hunk 4]
```diff
@@ -28,42 +18,16 @@ resource "github_membership" "brianhays" {
   role     = "member"
 }
 
-resource "github_membership" "Noah_Kantrowitz" {
-  username = "coderanger"
-  role     = "member"
-}
-
 resource "github_membership" "Dylan_Myers" {
```
Comment (JohnRoesler) [inline modules/org_membership/main.tf]: zero commits in our org

File: modules/org_membership/main.tf [hunk 5]
```diff
@@ -1,13 +1,3 @@
...
-  username = "atomic-penguin"
-  role     = "member"
-}
-
-resource "github_membership" "TD-4242" {
-  username = "TD-4242"
-  role     = "member"
-}
-
 resource "github_membership" "atrull" {
```
Comment (JohnRoesler) [inline modules/org_membership/main.tf]: zero commits in the last year

File: modules/org_membership/main.tf [hunk 6]
```diff
@@ -79,56 +43,21 @@ resource "github_membership" "Aaron_Kalin" {
   role     = "member"
 }
 
-resource "github_membership" "medyagh" {
```
Comment (JohnRoesler) [inline modules/org_membership/main.tf]: should also remove https://github.com/sous-chefs/terraform-github-org/blob/159310dd9218f094528b0fedeefd15bd4ec9b6c4/chef_de_partie.tf#L93-L97

**Pull request merged**